packages/serverless-orchestration/test/ServerlessHub.js

```diff
@@ -455,9 +455,9 @@ contract("ServerlessHub.js", function(accounts) {
     assert.isTrue(lastSpyLogIncludes(hubSpy, "error Command INVALID not found"));
     assert.isTrue(
       responseObject.output.errorOutputs["testServerlessMonitorError2"].execResponse.stderr.includes(
-        "Returned values aren't valid, did it run Out of Gas?"
+        "Contract code hash is null"
       )
     ); // invalid emp error
-    assert.isTrue(lastSpyLogIncludes(hubSpy, "Returned values aren't valid, did it run Out of Gas?"));
+    assert.isTrue(lastSpyLogIncludes(hubSpy, "Contract code hash is null"));
```

Comment:
@chrismaree not sure why I had to change these error messages. Looking at the monitor index.js, this seems right. Not sure how it previously produced the `Returned values aren't valid, did it run Out of Gas?` error since it first grabs the contract hash before making contract calls (I think),

Reply:
well the `Contract code hash is null` is the error you added to the `FindContractVersion` above with
```
  assert(contractCodeHash !== null, "Contract code hash is null");
```
right?

In any case this test is meant to validate that the hub correctly deals with an `0x0000000000000000000000000000000000000000` set as the contract address and the associated error produced within the spokes (and child process initialised by the spoke). Your change made to the `FindContractVersion` is catching _before_ it gets to the bot's initialisation, where it was previously rejecting. 

Strictly speaking, your change is _better_ than what it was doing before as it is erroring out when trying to fetch byte code.

i.e this is expected, given your code change and seems to be correct :)